.github/workflows/web-deploy.yml

```
@@ -1,62 +1,62 @@
-          # 環境設定
-          VITE_ENV: production
-          VITE_USE_MOCK: 'false'
-          
-      - name: Deploy to Firebase
-        uses: FirebaseExtended/action-hosting-deploy@v0
-        with:
-          repoToken: '${{ secrets.GITHUB_TOKEN }}'
-          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT }}'
-          channelId: live
-          projectId: ${{ secrets.FIREBASE_PROJECT_ID }}
+name: Deploy Web to Firebase Hosting
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)